format-all.el

```diff
@@ -991,7 +992,13 @@ Consult the existing formatters for examples of BODY."
     "format"
     (let ((config-file (format-all--locate-file ".formatter.exs")))
       (when config-file (list "--dot-formatter" config-file)))
-    "-")))
+    "--stdin-filename"
+    (cond ((buffer-file-name)
+       (list "--stdin-filename" (buffer-file-name) "-"))
+      ((equal language "HTML+EEX")
+       (list "--stdin-filename" "stdin.heex" "-"))
+      (t
+       (list))))))
```

Comment:
`(list)` simply returns `nil`, unsure of purpose but included per review

Reply:
Yes, it's just an empty list.

But if `-` is required for stdin, it shouldn't be empty... My mistake. I'll merge the PR and fix it.